### DISCUSSION TITLE
performance drops when fine-tuning resnet50 in siamrpn++

### DISCUSSION THREAD

**Discussion by WuFengGit** [STVIR/pysot#541] [Q&A]
Has anyone encountered this problem when training siamrpn++? Before fine-tuning the backbone network, the success rate of OTB2015 is 0.6. Once the backbone network is fine-tuned, it drops straight to 0.4. The training loss drops normally.
